### ISSUE TITLE
Commit 062ee99 breaks pip install SSL SSL: CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by arush** [tiangolo/uwsgi-nginx-docker#28]
Since commit 062ee99 (12 hours ago) our docker builds have been failing and i narrowed it down to this commit. I reverted the commit, published a new image: https://hub.docker.com/r/trycom/uwsgi-nginx-docker/tags/ and used that instead and everything worked as normal.

To reproduce, simply create a Dockerfile

```
FROM tiangolo/uwsgi-nginx:python3.6
RUN pip install requests=2.11.1
RUN pip install googlemaps
```

and you'll get the error when attempting to pip install googlemaps. Yesterday this used to work, so having published a reverted image you can do the following to see expected results:

```
FROM trycom/uwsgi-nginx-docker:39f6fdb
RUN pip install requests=2.11.1
RUN pip install googlemaps
```

Also, as a separate comment, you should tag your docker images with the commit so we dont have to publish our own reverted versions

**Comment by tiangolo** [comment]
I got errors with both versions.
```
Invalid requirement: 'requests=2.11.1'
= is not a valid operator. Did you mean == ?
```

Then I changed the requests requirement to `requests==2.11.1` and it worked. In both versions.

I've not been able to reproduce the SSL issue, even though @desaintmartin seems to have it too in #25. 

Can you try doing an explicit `docker pull tiangolo/uwsgi-nginx:python3.6` before doing the build and see if that changes anything? I've seen problems with building images where the building server keeps an old version of an image and bases on top of it.

In any case, if you can find a way for me to reproduce the issue that would be great. Because I haven't been able to see it first hand.

**Comment by babaMar** [comment]
Maybe related, I'm getting an SSL problem when building an image with private dependencies:
```
fatal: unable to access 'https://<auth-token>@github.com/private/repo.git/': Problem with the SSL CA cert (path? access rights?)
```
```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/basecommand.py", line 228, in main
    status = self.run(options, args)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/commands/install.py", line 291, in run
    resolver.resolve(requirement_set)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/resolve.py", line 103, in resolve
    self._resolve_one(requirement_set, req)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/resolve.py", line 257, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/resolve.py", line 194, in _get_abstract_dist_for
    req, self.require_hashes, self.use_user_site, self.finder,
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/operations/prepare.py", line 347, in prepare_editable_requirement
    req.update_editable(not self._download_should_save)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/req/req_install.py", line 633, in update_editable
    vcs_backend.obtain(self.source_dir)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/vcs/git.py", line 187, in obtain
    self.run_command(['clone', '-q', url, dest])
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/vcs/__init__.py", line 427, in run_command
    spinner=spinner)
  File "/usr/local/lib/python3.6/site-packages/pip/_internal/utils/misc.py", line 698, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "git clone -q https://<auth-token>@github.com/private/repo.git /usr/local/src/repo" failed with error code 128 in None
```

**Comment by jkatzhrs** [comment]
We solved this by adding `apt-get update && apt-get install -y apt-transport-https ca-certificates` to our docker file. You should never remove `ca-certificates` from a distro but it seems like it was done by mistake on https://github.com/tiangolo/uwsgi-nginx-docker/commit/062ee99240908b57b759f240b248006e4c822e43#diff-0aaae9c4760d1637b13a4453d8d327f3R82

**Comment by babaMar** [comment]
@jkatzhrs thanks for that, that solved my problem as well

**Comment by tiangolo** [comment]
Thanks @jkatzhrs , that makes sense. Actually, all that block is copied directly from the official Nginx image. I try to keep this image as close as possible to the official Python image mixed with the official Nginx image.

But I don't know why the official Nginx removes that: https://github.com/nginxinc/docker-nginx/blob/590f9ba27d6d11da346440682891bee6694245f5/mainline/stretch/Dockerfile#L83

I'll check your PR right away.

**Comment by tiangolo** [comment]
This should be fixed by PR #29 by @jkatzhrs , please check if it works for you. The images are being built right now, they should be available in some minutes.

**Comment by tiangolo** [comment]
As there were no more comments I'll assume the issue is solved and close it.
